docs-core/src/main/java/com/sismics/docs/core/model/jpa/Review.java

```diff
@@ -0,0 +1,102 @@
+package com.sismics.docs.core.model.jpa;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * Link between an user and a group.
+ * 
+ * @author jessedin
+ */
+@Entity
+@Table(name = "T_REVIEW")
+public class Review {
```

Comment:
One more thing: let's add another column that represents the ID of the review itself (i.e. a primary key). That one should be the one with the @Id decorator.